arma/server/src/lib.rs

```diff
@@ -56,9 +56,13 @@ fn init() -> Extension {
         .group("garage", commands::garage::group())
         .group("gear", commands::gear::group())
         .group("log", commands::log::group())
-        .group("reputation", commands::reputation::group())
-        .state(commands::garage::PendingSpawn::default())
-        .finish();
+        .state(commands::garage::PendingSpawn::default());
+
+    if *SERVER_ID == "primary-main" {
+        ext.group("reputation", commands::reputation::group());
```

Comment:
It should ignore the command instead of disabling it, Arma will still try to call this

Reply:
updated